### PR TITLE
Adjust service-catalog and content views for ui-api modularisation

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Adjust service-catalog and content views for ui-api modularization:
  - Hide `Bound Applications` of table in core instances view if `servicecatalogaddons` backend module is disabled.
  - Hide `Bound Applications` tab in details instance view if `servicecatalogaddons` backend module is disabled.
  - Hide documentation of service class in appropriate views if `content` backend module is disabled.
  - Show message if appropriate backend module (`servicecatalog` and `content`) is disabled.
- Bump dependencies in all views of apollo client (for working `errorPolicy`).

**Related issue(s)**
See also https://github.com/kyma-project/kyma/issues/1883

### CODE REVIEW

File: brokers/src/components/App/App.component.js [hunk 1]
```diff
@@ -39,9 +43,13 @@ class App extends React.Component {
...
         <div className="ph3 pv1 background-gray">
-          <Switch>
-            <Route exact path="/" component={ServiceBrokers} />
-          </Switch>
+          {backendModuleExists("servicecatalog") ? (
+            <Switch>
+              <Route exact path="/" component={ServiceBrokers} />
+            </Switch>
+          ) : (
+            <BackendModuleNotExists>Service Catalog backend module is disabled.</BackendModuleNotExists>
```
Comment (pkosiec) [inline brokers/src/components/App/App.component.js]: Pass just "service catalog" through props

File: brokers/src/commons/helpers.js [hunk 2]
```diff
@@ -8,3 +10,7 @@ export const statusColor = statusType => {
       return '#ffb600';
   }
 };
+
+export const backendModuleExists = (name) => {
+  return builder.getBackendModules().some(mod => mod === name);
```
Comment (pkosiec) [inline brokers/src/commons/helpers.js]: I would rather use `find` 😛

File: brokers/src/store/transformers.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  parseValue(value) {
+    if (typeof value !== 'string') {
+      return value;
+    }
+
+    return JSON.parse(value);
+  },
+};
+
+const getTransformers = fieldTransformer => ({
```
Comment (pkosiec) [inline brokers/src/store/transformers.js]: Is it used?

File: brokers/src/store/transformers.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+});
+
+export const getLinkTransformers = () => {
+  return getTransformers(ScalarSerializer);
+};
+
+export const transformDataScalarStringsToObjects = data => {
+  if (!data) {
+    return data;
```
Comment (pkosiec) [inline brokers/src/store/transformers.js]: `return undefined`?

File: brokers/src/store/transformers.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  }
+
+  const transformers = getTransformers(ScalarDeserializer);
+  parseObject(transformers, data);
+  return data;
+};
+
+export const transformDataScalarObjectsToStrings = data => {
+  if (!data) {
```
Comment (pkosiec) [inline brokers/src/store/transformers.js]: `return undefined`?

File: catalog/src/components/App/App.component.js [hunk 6]
```diff
@@ -70,10 +72,14 @@ class App extends React.Component {
...
-            <Route exact path="/" component={MainPage} />
-            <Route exact path="/details/:name" component={InstanceDetails} />
-          </Switch>
+          {backendModuleExists("servicecatalog") ? (
+            <Switch>
+              <Route exact path="/" component={MainPage} />
+              <Route exact path="/details/:name" component={InstanceDetails} />
+            </Switch>
+          ) : (
+            <BackendModuleNotExists>Service Catalog backend module is disabled.</BackendModuleNotExists>
```
Comment (pkosiec) [inline catalog/src/components/App/App.component.js]: Pass only module name to the component

File: components/react/src/components/BackendModuleNotExists/index.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { Panel, PanelBody } from "fundamental-react";
+
+import {
+    Wrapper
+} from './styled';
+
+const BackendModuleNotExists = ({ children }) => (
```
Comment (pkosiec) [inline components/react/src/components/BackendModuleNotExists/index.js]: Rename to `BackendModuleDisabled`

File: content/src/store/transformers.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+  parseValue(value) {
+    if (typeof value !== 'string') {
+      return value;
+    }
+
+    return JSON.parse(value);
+  },
+};
+
+const getTransformers = fieldTransformer => ({
```
Comment (pkosiec) [inline content/src/store/transformers.js]: Remove empty function

File: brokers/src/commons/helpers.js (hunk 2 above)
Comment (magicmatatjahu) [reply brokers/src/commons/helpers.js]: Find return element and I need only true or false.

File: brokers/src/components/App/App.component.js (hunk 1 above)
Comment (magicmatatjahu) [reply brokers/src/components/App/App.component.js]: I thought about that, but please look at message for `content-ui`, so passing only name of module is wrong idea.

File: brokers/src/store/transformers.js (hunk 3 above)
Comment (magicmatatjahu) [reply brokers/src/store/transformers.js]: This is not used, but I left `transformers.js` for future development (this is copied from instances state).

File: components/react/src/components/BackendModuleNotExists/index.js (hunk 7 above)
Comment (magicmatatjahu) [reply components/react/src/components/BackendModuleNotExists/index.js]: Hmm... ok, better name 😄

File: brokers/src/components/App/App.component.js (hunk 1 above)
Comment (pkosiec) [reply brokers/src/components/App/App.component.js]: I would rather unify messaging 😛

File: brokers/src/commons/helpers.js (hunk 2 above)
Comment (pkosiec) [reply brokers/src/commons/helpers.js]: Sorry, I meant `includes` 😛

File: catalog/src/store/transformers.js [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+});
+
+export const getLinkTransformers = () => {
+  return getTransformers(ScalarSerializer);
+};
+
+export const transformDataScalarStringsToObjects = data => {
+  if (!data) {
+    return data;
```
Comment (pkosiec) [inline catalog/src/store/transformers.js]: `return undefined`

File: content/src/store/transformers.js [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+  
+});
+
+export const getLinkTransformers = () => {
+  return getTransformers(ScalarSerializer);
+};
+
+export const transformDataScalarStringsToObjects = data => {
+  if (!data) {
```
Comment (pkosiec) [inline content/src/store/transformers.js]: `return undefined`

File: content/src/store/transformers.js [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+  }
+
+  const transformers = getTransformers(ScalarDeserializer);
+  parseObject(transformers, data);
+  return data;
+};
+
+export const transformDataScalarObjectsToStrings = data => {
+  if (!data) {
+    return data;
```
Comment (pkosiec) [inline content/src/store/transformers.js]: `return undefined`

**Pull request merged**